### PR TITLE
FIX: Set the first launch flag

### PR DESCRIPTION
**Pull request by marcosrdz**

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/9gxyfnued20cq88pnqve80532m

[download apk](http://filestorage.bluewallet.io:1488/1591104206.apk)

File: screen/wallets/walletMigrate.js [hunk 1]
```diff
@@ -12,12 +12,14 @@ export default class WalletMigrate {
 
   // 0: Let's start!
   async start() {
-    const firstLaunch = await AsyncStorage.getItem('RnSksIsAppInstalled');
-    if (firstLaunch === undefined || firstLaunch === null || firstLaunch === false || firstLaunch === '') {
+    const isNotFirstLaunch = await AsyncStorage.getItem('RnSksIsAppInstalled');
+    if (!isNotFirstLaunch) {
       try {
+        console.warn('It is the first launch...')
```
Comment (Overtorment) [inline screen/wallets/walletMigrate.js:18]: this warn will probably annoy us a lot when working in simulators. oi wei, will remove later

File: screen/wallets/walletMigrate.js [hunk 2]
```diff
@@ -12,12 +12,14 @@ export default class WalletMigrate {
...
         await RNSecureKeyStore.setResetOnAppUninstallTo(false);
         const deleteWalletsFromKeychain = await RNSecureKeyStore.get(AppStorage.DELETE_WALLET_AFTER_UNINSTALL);
         await RNSecureKeyStore.setResetOnAppUninstallTo(deleteWalletsFromKeychain === '1');
+        await AsyncStorage.setItem('RnSksIsAppInstalled', '1');
       } catch (_e) {}
       await AsyncStorage.setItem('RnSksIsAppInstalled', '1');
```
Comment (Overtorment) [inline screen/wallets/walletMigrate.js:24]: is that correct, there should be 2 lines setting RnSksIsAppInstalled, 22 & 24 ?

Comment (marcosrdz) [reply screen/wallets/walletMigrate.js:24]: Line 22 is after keychain logic rubs.

Lie 24 is if the async getitem fails.

File: screen/wallets/walletMigrate.js (hunk 1 above)
Comment (marcosrdz) [reply screen/wallets/walletMigrate.js:18]: Only runs on first launch. you would have to reset your simulator to see it all the time.

**Pull request merged**